src/main/java/org/store/book/OwnerDTO.java

```diff
@@ -0,0 +1,35 @@
+package org.store.book;
+
+public class OwnerDTO {
```

Comment:
Unnecessary class as the User already has these details. Every user can be a owner or borrower. Keep only one class.